django/contrib/staticfiles/apps.py

```diff
@@ -5,3 +5,4 @@
 class StaticFilesConfig(AppConfig):
     name = 'django.contrib.staticfiles'
     verbose_name = _("Static Files")
+    ignore_patterns = ['CVS', '.*', '*~']
```

Comment:
Maybe `collectstatic_ignore_patterns`? I think the usage location of the setting isn't intuitive as currently named.